tox.ini

```diff
@@ -57,6 +60,7 @@ commands =
     mypy-focal: mypy --python-version 3.7 uaclient/ features/ lib/
     black: black --check --diff uaclient/ features/ lib/ setup.py
     isort: isort --check --diff uaclient/ features/ lib/ setup.py
+    shellcheck: bash ./shell_lint.sh
```

Comment:
+1 on this, but:
* `{ }` sequences will not be expanded because tox doesn't execute the command in a shell. We need `sh -c` there.
* Do we actually need those `-P` flags? AIUI `-P` is meant to specify paths to search when shellcheck encounters a `source` (or `.`) command.

Also: merging this will cause CI to fail, right? There are several scripts raising shellcheck warnings/errors.

Reply:
+1 to everything.

Currently ci specifically runs flake8, black, isort, and mypy. So this shouldn't break CI as written AFAICT.

We also generally won't want to fix some things. E.g. it'd be a waste of time to fix old sru test scripts, so we probably shouldn't run shellcheck on those at all.

Reply:
Interestingly, `sh -c` does not work and results in the linting failing.However, `bash -c` works fine even without the **allowlist_externals** config.

Reply:
Right, `{ }` sequence expansion is bash only syntax. In Ubuntu we can just assume bash to be there. I do however get:
```
WARNING: test command found but not installed in testenv
  cmd: /usr/bin/sh
  env: /home/paride/ubuntu/git/ubuntu-advantage-client/.tox/shellcheck
Maybe you forgot to specify a dependency? See also the allowlist_externals envconfig setting.
```

Reply:
Good catch. It looks like you'll need `allowlist_externals = /usr/bin/bash` to get rid of that warning @CalvoM

Reply:
Yes, let me add it.